plugins/wasi_nn/ggml.cpp

```diff
@@ -522,7 +602,47 @@ Expect<ErrNo> setInput(WasiNNEnvironment &Env, uint32_t ContextId,
   const bool AddBos = llama_should_add_bos_token(GraphRef.LlamaModel);
   const std::string Prompt(reinterpret_cast<char *>(Tensor.Tensor.data()),
                            Tensor.Tensor.size());
-  CxtRef.LlamaInputs = llama_tokenize(LlamaContext, Prompt, AddBos, true);
+  if (GraphRef.MMProjModelPath == ""sv || GraphRef.ImagePath == ""sv) {
+    // Text only prompt.
+    CxtRef.LlamaInputs = llama_tokenize(LlamaContext, Prompt, AddBos, true);
+  } else {
+    // Handle llava format prompt.
+    // We split prompt by {WASMEDGE_IMAGE} as placeholder and save the position.
+    const std::string_view PromptPlaceholder = "{WASMEDGE_IMAGE}";
```

Comment:
Let's use `<image>`, the same as llama.cpp

Reply:
Updated.